tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "@ethereumjs/config-tsc",
+  "include": ["src/**/*.ts", "test/**/*.ts", "examples/**/*.ts"],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "*": ["definitions/*"]
+    },
+    "typeRoots": ["definitions", "node_modules/@types"],
+    "declaration": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "downlevelIteration": true
+  }
+}
```

Comment:
This needs an update as discussed. If this causes too much roundup work I would be also ok with having this adopted to the standard of discussion and left here for now. We could then integrate with the config library later on.

Reply:
I'll update the corresponding build projects with this new settings.

Reply:
Here is the corresponding PR - https://github.com/ethereumjs/ethereumjs-config/pull/23